scripts/gen-edpm-kustomize.sh

```diff
@@ -0,0 +1,111 @@
+. ${SCRIPTPATH}/common.sh --source-only
+
+if [ -z "$NAMESPACE" ]; then
+    echo "Please set NAMESPACE"; exit 1
+fi
+
+if [ -z "$KIND" ]; then
+    echo "Please set SERVICE"; exit 1
+fi
+
+if [ -z "$DEPLOY_DIR" ]; then
+    echo "Please set DEPLOY_DIR"; exit 1
+fi
+
+NAME=${KIND,,}
+
+if [ ! -d ${DEPLOY_DIR} ]; then
+    mkdir -p ${DEPLOY_DIR}
+fi
+
+pushd ${DEPLOY_DIR}
+
+cat <<EOF >kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+namespace: ${NAMESPACE}
+patches:
+- target:
+    kind: ${KIND}
+  patch: |-
+    - op: replace
+      path: /spec/deployStrategy/deploy
+      value: true
+    - op: replace
+      path: /spec/nodes/edpm-compute-0/ansibleHost
+      value: ${EDPM_COMPUTE_NODE0_IP}
+    - op: replace
+      path: /spec/nodes/edpm-compute-0/openStackAnsibleEERunnerImage
+      value: ${OPENSTACK_RUNNER_IMG}
+    - op: replace
+      path: /spec/nodes/edpm-compute-0/node/ansibleSSHPrivateKeySecret
+      value: ${EDPM_ANSIBLE_SECRET}
+    - op: replace
+      path: /spec/nodes/edpm-compute-0/node/ansibleVars
+      value: |
+        tenant_ip: ${EDPM_COMPUTE_NODE0_IP}
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/ansibleHost
+      value: ${EDPM_COMPUTE_NODE1_IP}
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/openStackAnsibleEERunnerImage
+      value: ${OPENSTACK_RUNNER_IMG}
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/node/ansibleSSHPrivateKeySecret
+      value: ${EDPM_ANSIBLE_SECRET}
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/node/ansibleVars
+      value: |
+        tenant_ip: ${EDPM_COMPUTE_NODE1_IP}
+    - op: replace
+      path: /spec/roles/edpm-compute/openStackAnsibleEERunnerImage
+      value: ${OPENSTACK_RUNNER_IMG}
+    - op: replace
+      path: /spec/roles/edpm-compute/nodeTemplate/ansibleVars
+      value: |
+        edpm_sshd_allowed_ranges: ${EDPM_SSHD_ALLOWED_RANGES}
+        edpm_network_config_template: ${EDPM_NETWORK_CONFIG_TEMPLATE}
+        edpm_network_config_hide_sensitive_logs: false
+        neutron_physical_bridge_name: br-ex
+        neutron_public_interface_name: eth0
+        ctlplane_dns_nameservers:
+        - 192.168.122.1
+        dns_search_domains: []
+        edpm_ovn_dbs: 192.168.24.1
+        edpm_ovn_metadata_agent_default_transport_url: ${EDPM_OVN_METADATA_AGENT_TRANSPORT_URL}
+        edpm_ovn_metadata_agent_metadata_agent_ovn_ovn_sb_connection: ${EDPM_OVN_METADATA_AGENT_SB_CONNECTION}
+        edpm_ovn_metadata_agent_metadata_agent_default_nova_metadata_host: ${EDPM_OVN_METADATA_AGENT_NOVA_METADATA_HOST}
+        edpm_ovn_metadata_agent_metadata_agent_default_metadata_proxy_shared_secret: ${EDPM_OVN_METADATA_AGENT_PROXY_SHARED_SECRET}
+        edpm_ovn_metadata_agent_default_bind_host: ${EDPM_OVN_METADATA_AGENT_BIND_HOST}
+        edpm_chrony_ntp_servers:
+        - ${EDPM_CHRONY_NTP_SERVER}
+        edpm_ovn_dbs:
+        - ${EDPM_OVN_DBS}
```

Comment:
This make `make edpm_deploy` fail with:
```
2023-03-21T12:44:29+01:00	INFO	Reconciling	{"controller": "openstackdataplanerole", "controllerGroup": "dataplane.openstack.org", "controllerKind": "OpenStackDataPlaneRole", "OpenStackDataPlaneRole": {"name":"edpm-compute","namespace":"openstack"}, "namespace": "openstack", "name": "edpm-compute", "reconcileID": "774a30cb-737c-4eea-9e67-7798438e3241"}
2023-03-21T12:44:29+01:00	INFO	Adding label openstackdataplane=openstack-edpm	{"controller": "openstackdataplanerole", "controllerGroup": "dataplane.openstack.org", "controllerKind": "OpenStackDataPlaneRole", "OpenStackDataPlaneRole": {"name":"edpm-compute","namespace":"openstack"}, "namespace": "openstack", "name": "edpm-compute", "reconcileID": "774a30cb-737c-4eea-9e67-7798438e3241"}
2023-03-21T12:44:29+01:00	ERROR	Unable to generate inventory for edpm-compute	{"controller": "openstackdataplanerole", "controllerGroup": "dataplane.openstack.org", "controllerKind": "OpenStackDataPlaneRole", "OpenStackDataPlaneRole": {"name":"edpm-compute","namespace":"openstack"}, "namespace": "openstack", "name": "edpm-compute", "reconcileID": "774a30cb-737c-4eea-9e67-7798438e3241", "ObjectType": "*v1beta1.OpenStackDataPlaneRole", "ObjectNamespace": "openstack", "ObjectName": "edpm-compute", "error": "yaml: unmarshal errors:\n  line 17: mapping key \"edpm_ovn_dbs\" already defined at line 9"}
github.com/openstack-k8s-operators/lib-common/modules/common/util.LogErrorForObject
	/home/gibi/go/pkg/mod/github.com/openstack-k8s-operators/lib-common/modules/common@v0.0.0-20230307144813-39ed0da90b23/util/log.go:65
github.com/openstack-k8s-operators/dataplane-operator/controllers.(*OpenStackDataPlaneRoleReconciler).Reconcile
	/home/gibi/upstream/git/openstack-k8s-operators/dataplane-operator/controllers/openstackdataplanerole_controller.go:139
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile
	/home/gibi/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.5/pkg/internal/controller/controller.go:122
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
	/home/gibi/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.5/pkg/internal/controller/controller.go:323
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
	/home/gibi/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.5/pkg/internal/controller/controller.go:274
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
	/home/gibi/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.5/pkg/internal/controller/controller.go:235
```

Reply:
yeah, I accidentally had a duplicate entry, let me push the fix